Core/src/org/sleuthkit/autopsy/datasourcesummary/ui/DataSourceSummaryTabbedPane.java

```diff
@@ -138,10 +132,46 @@ public Runnable getOnClose() {
      * Creates new form TabPane
      */
     public DataSourceSummaryTabbedPane() {
+        this.tabs = getTabs();
         initComponents();
         postInit();
     }
 
+    /**
+     * Sends event that parent should close.
+     */
+    private void notifyParentClose() {
+        if (notifyParentClose != null) {
+            notifyParentClose.run();
+        }
+    }
+
+    /**
+     * Sets the listener for parent close events.
+     *
+     * @param parentCloseAction The observer.
+     */
+    void setParentCloseListener(Runnable parentCloseAction) {
+        notifyParentClose = parentCloseAction;
+    }
+
+    private List<DataSourceTab> getTabs() {
+        List<DataSourceTab> tabs = Arrays.asList(
```

Comment:
If this method is intended to improve readability or for some other reason it is certainly fine, but we should avoid hiding the tabs field with a local variable named tabs.